streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsRebalanceListener.java

```diff
@@ -54,6 +54,9 @@ public void onPartitionsAssigned(final Collection<TopicPartition> partitions) {
         if (assignmentErrorCode.get() == AssignorError.INCOMPLETE_SOURCE_TOPIC_METADATA.code()) {
             log.error("Received error code {}", assignmentErrorCode.get());
             throw new MissingSourceTopicException("One or more source topics were missing during rebalance");
+        }else if(assignmentErrorCode.get() == AssignorError.SHUTDOWN_REQUESTED.code()){
+            //throw new ShutdownRequestedException("onPartition assigned"); //TODO: receive request and call requestClose()
+            //requestClose();
```

Comment:
until the shut down application form the stream thread functionality is added this will not shutdown the app